demos/entityscroller/entityscroller.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"image/color"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+	"engo.io/engo/common"
+)
+
+type GameWorld struct{}
+
+type Character struct {
+	ecs.BasicEntity
+	common.RenderComponent
+	common.SpaceComponent
+}
+
+type Tile struct {
+	ecs.BasicEntity
+	common.RenderComponent
+	common.SpaceComponent
+}
+
+type ControlSystem struct {
+	entity *Character
+}
+
+func (c *ControlSystem) Add(char *Character) {
+	c.entity = char
+}
+
+func (c *ControlSystem) Remove(ecs.BasicEntity) {
+	c.entity = nil
```

Comment:
This is not good practice, at all. 

Note that `Remove` will get called as soon as *any* entity gets removed from *any* system. Even though it'll work in this scenario (because nothing is ever removed), this is not be what we should teach others. 

```go
func (c *ControlSystem) Remove(basic ecs.BasicEntity) {
	if c.entity != nil && basic.ID() == c.entity.ID() {
        c.entity = nil
    }
}
```

This would work, though. (given it compiles, haven't checked syntax)